locales/it/index.yml

```diff
@@ -2598,7 +2598,7 @@ bonus:
         sport: sport
         health: salute e benessere
     cta:
-      activeBonus: Attiva la tua Carta Giovani
+      activeBonus: Richiedi la tua CGN
```

Comment:
@AliceAzzolini this CTA needs some love :)

Reply:
```suggestion
      activeBonus: Attiva la Carta
```